### PR TITLE
docs(benchmark): add info about binary packages and docker images

### PR DESCRIPTION
**Pull request by id**
Fixes https://emqx.atlassian.net/browse/EMQX-8084

### CODE REVIEW

**Comment by wivwiv** [comment]
@LenaLenaPan 
Please provide the translation in Chinese. Also, we will upgrade emqx.com this Friday to provide a download of emqtt-bench, which you can add to the documentation.

The expected download address
https://www.emqx.com/en/try?product=neuron

File: en_US/performance/benchmark.md [hunk 1]
```diff
@@ -2,9 +2,46 @@
 
 [emqtt_bench](https://github.com/emqx/emqtt_bench) is a concise and powerful MQTT protocol performance testing tool written with Erlang. If you need testing services with large-scale scenarios and in-depth customization , the test service [XMeter](https://www.xmeter.net/) is recommended.
 
-## Compile and install
+## Docker image
```
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
## Install Test Tool
### Run Docker Image
```

File: en_US/performance/benchmark.md [hunk 2]
```diff
@@ -2,9 +2,46 @@
...
+`emqtt_bench` docker images are pushed to [hub.docker.com](https://hub.docker.com/r/emqx/emqtt-bench/tags), and `:latest` tag is updated with each new version:
+
+```bash
+docker run -it emqx/emqtt-bench:latest
+Usage: emqtt_bench pub | sub | conn [--help]
+```
+
+Note that docker image name is using hyphen '-', while binary script name is with underscore '_'.
+
+## Binary package
```
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
### Install Binary Package
```

File: en_US/performance/benchmark.md [hunk 3]
```diff
@@ -2,9 +2,46 @@
...
+
+- CentOS 7
+- Rocky Linux 8
+- Debian 9
+- Debain 10
+- Ubuntu 16.04
+- Ubuntu 18.04
+- Ubuntu 20.04
+
+Please check them out on the [Releases](https://github.com/emqx/emqtt-bench/releases) page.
```
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
For detailed information on each release, see [Releases](https://github.com/emqx/emqtt-bench/releases).
```

File: en_US/performance/benchmark.md [hunk 4]
```diff
@@ -2,9 +2,46 @@
...
+```bash
+docker run -it emqx/emqtt-bench:latest
+Usage: emqtt_bench pub | sub | conn [--help]
+```
+
+Note that docker image name is using hyphen '-', while binary script name is with underscore '_'.
+
+## Binary package
+
+We create binary packages with each release for the following platforms:
```
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
You can download the released binary packages and install the `emqtt_bench` on the following platforms:
```

File: en_US/performance/benchmark.md [hunk 5]
```diff
@@ -2,9 +2,46 @@
...
+- Rocky Linux 8
+- Debian 9
+- Debain 10
+- Ubuntu 16.04
+- Ubuntu 18.04
+- Ubuntu 20.04
+
+Please check them out on the [Releases](https://github.com/emqx/emqtt-bench/releases) page.
+
+Here is how to install `emqtt_bench` on, for example, Ubuntu 20.04:
```
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
For example, here is how to install `emqtt_bench` on Ubuntu 20.04:
```

File: en_US/performance/benchmark.md [hunk 6]
```diff
@@ -2,9 +2,46 @@
...
+mkdir emqtt_bench && cd emqtt_bench
+wget https://github.com/emqx/emqtt-bench/releases/download/0.4.8/emqtt-bench-0.4.8-ubuntu20.04-amd64.tar.gz
+tar xfz emqtt-bench-0.4.8-ubuntu20.04-amd64.tar.gz
+rm emqtt-bench-0.4.8-ubuntu20.04-amd64.tar.gz
+
+./emqtt_bench
+Usage: emqtt_bench pub | sub | conn [--help]
+```
+
+## Build from source
```
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
### Build from Source
```

File: en_US/performance/benchmark.md [hunk 7]
```diff
@@ -2,9 +2,46 @@
 
 [emqtt_bench](https://github.com/emqx/emqtt_bench) is a concise and powerful MQTT protocol performance testing tool written with Erlang. If you need testing services with large-scale scenarios and in-depth customization , the test service [XMeter](https://www.xmeter.net/) is recommended.
 
-## Compile and install
+## Docker image
 
-The operation of `emqtt_bench` depends on the operating environment of Erlang/OTP 21.2 and above version. The installation process is skipped. For details, please refer to the online installation tutorials.
+`emqtt_bench` docker images are pushed to [hub.docker.com](https://hub.docker.com/r/emqx/emqtt-bench/tags), and `:latest` tag is updated with each new version:
```
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
You can install the benchmark tool by running the `emqtt_bench` docker image pushed to [hub.docker.com](https://hub.docker.com/r/emqx/emqtt-bench/tags). The `:latest` tag is updated with each new version:
```

**Comment by Meggielqk** [comment]
Should the title be changed to "Benchmark Test (eMQTT-Bench)"? @wivwiv @id

File: en_US/performance/benchmark.md [hunk 8]
```diff
@@ -1,10 +1,47 @@
 # Benchmark
 
-[emqtt_bench](https://github.com/emqx/emqtt_bench) is a concise and powerful MQTT protocol performance testing tool written with Erlang. If you need testing services with large-scale scenarios and in-depth customization , the test service [XMeter](https://www.xmeter.net/) is recommended.
+[eMQTT-Bench](https://www.emqx.com/en/try?product=emqtt-bench) is a concise and powerful MQTT protocol benchmark testing tool written with Erlang. If you need testing services with large-scale scenarios and in-depth customization , the test service [XMeter](https://www.xmeter.net/) is recommended.
```
Comment (id) [inline en_US/performance/benchmark.md]: "performance testing" is more correct than "benchmark testing".
"Benchmark" word by itself means "performance testing".

If you want to use "Benchmark" here, I would recommend to just say "MQTT protocol benchmark tool"

File: en_US/performance/benchmark.md (hunk 1 above)
Comment (id) [reply en_US/performance/benchmark.md]: Docker image is one of the options to install emqtt-bench, another one is by downloading binary package (below).
If we say here "Install Test Tool" we should say the same down below as well. Both options are equivalent.

File: en_US/performance/benchmark.md (hunk 6 above)
Comment (id) [reply en_US/performance/benchmark.md]: Building from source is yet another option of installing emqtt-bench, equivalent to docker image and binary package.

File: en_US/performance/benchmark.md [hunk 9]
```diff
@@ -1,10 +1,47 @@
...
+tar xfz emqtt-bench-0.4.8-ubuntu20.04-amd64.tar.gz
+rm emqtt-bench-0.4.8-ubuntu20.04-amd64.tar.gz
+
+./emqtt_bench
+Usage: emqtt_bench pub | sub | conn [--help]
+```
+
+## Build from source
+
+The operation of `emqtt_bench` depends on the operating environment of [Erlang/OTP](https://www.erlang.org/) 21.2 and above version. The installation process is skipped. For details, please refer to the online installation tutorials.
```
Comment (id) [inline en_US/performance/benchmark.md]:
```suggestion
`emqtt_bench` is written in Erlang and requires [Erlang/OTP](https://www.erlang.org/) 21.2 and above version to build it. The installation process of Erlang/OTP is skipped. For details, please refer to the online installation tutorials.
```

File: en_US/performance/benchmark.md (hunk 8 above)
Comment (Meggielqk) [inline en_US/performance/benchmark.md]:
```suggestion
[eMQTT-Bench](https://www.emqx.com/en/try?product=emqtt-bench) is a concise and powerful MQTT protocol benchmark tool written with Erlang. If you need testing services with large-scale scenarios and in-depth customization , the test service [XMeter](https://www.xmeter.net/) is recommended.
```

Comment (Meggielqk) [reply en_US/performance/benchmark.md]: Ok, I changed it to "benchmark tool" to be consistent with the saying on our offical website. See the change in the "Suggested change" below.

File: en_US/performance/benchmark.md (hunk 1 above)
Comment (Meggielqk) [reply en_US/performance/benchmark.md]: I unserstand there are thress options to install emqtt-bench. My previous suggestion to make the structure looks like the following. What do you think?

- Install Test Tool
   - Run Docker Image
   - Download Binary Package
   - Build from Source
- Use Test Tool
   - Publish
   - Subscribe
   - Connect
   - ...
- Typical Stress Test Scenario
   - ...

**Comment by id** [comment]
Thanks @Meggielqk !

**Comment by id** [comment]
Merged release-5.0 to resolve conflicts

File: zh_CN/performance/benchmark.md [hunk 10]
```diff
@@ -1,8 +1,59 @@
 # 性能测试
 
-[emqtt_bench](https://github.com/emqx/emqtt_bench) 是基于 Erlang 编写的，一个简洁强大的 MQTT 协议性能测试工具，如需大规模场景、深度定制化的测试服务推荐使用 [XMeter](https://www.emqx.com/zh/products/xmeter) 进行测试。
+当你部署了单个EMQX服务器或者建立了EMQX集群，你可以进行对你的部署进行性能测试以了解系统的能力。这个章节介绍了如何安装和使用[eMQTT-Bench](https://www.emqx.com/zh/try?product=emqtt-bench) 来进行性能测试。eMQTT-Bench是基于 Erlang 编写的，一个简洁强大的 MQTT 协议基准测试工具。如需大规模场景、深度定制化的测试服务推荐使用 [XMeter](https://www.emqx.com/zh/products/xmeter) 进行测试。
```
Comment (LenaLenaPan) [inline zh_CN/performance/benchmark.md]:
```suggestion
当你部署了单个 EMQX 服务器或者建立了 EMQX 集群，你可以进行对你的部署进行性能测试以了解系统的能力。这个章节介绍了如何安装和使用 [eMQTT-Bench](https://www.emqx.com/zh/try?product=emqtt-bench) 来进行性能测试。eMQTT-Bench是基于 Erlang 编写的，一个简洁强大的 MQTT 协议基准测试工具。如需大规模场景、深度定制化的测试服务推荐使用 [XMeter](https://www.emqx.com/zh/products/xmeter) 进行测试。
```

File: zh_CN/performance/benchmark.md [hunk 11]
```diff
@@ -1,8 +1,59 @@
...
 
-## 编译安装
+## 安装eMQTT-Bench
+
+你可以通过3种方式来安装eMQTT-Bench：
```
Comment (LenaLenaPan) [inline zh_CN/performance/benchmark.md]:
```suggestion
你可以通过 3 种方式来安装 eMQTT-Bench：
```

File: zh_CN/performance/benchmark.md [hunk 12]
```diff
@@ -1,8 +1,59 @@
...
+
+- 运行Docker镜像
```
Comment (LenaLenaPan) [inline zh_CN/performance/benchmark.md]:
```suggestion
- 运行 Docker 镜像
```

File: zh_CN/performance/benchmark.md [hunk 13]
```diff
@@ -1,8 +1,59 @@
...
+- 下载并安装二进制包
+- 从源代码构建
+
+### 运行Docker镜像
```
Comment (LenaLenaPan) [inline zh_CN/performance/benchmark.md]:
```suggestion
### 运行 Docker 镜像
```

File: zh_CN/performance/benchmark.md [hunk 14]
```diff
@@ -1,8 +1,59 @@
...
+
+你可以通过运行`emqtt_bench`镜像来安装测试工具。`emqtt_bench` docker 镜像已推送到 [hub.docker.com](https://hub.docker.com/r/emqx/emqtt-bench/tags), 且每个新版本都会更新`:latest`标签：
+
+```bash
+docker run -it emqx/emqtt-bench:latest
+Usage: emqtt_bench pub | sub | conn [--help]
+```
+
+注意，Docker镜像名称使用连字符“-”，而二进制脚本名称使用下划线“_”。
```
Comment (LenaLenaPan) [inline zh_CN/performance/benchmark.md]:
```suggestion
注意，Docker 镜像名称使用连字符“-”，而二进制脚本名称使用下划线“_”。
```

File: zh_CN/performance/benchmark.md [hunk 15]
```diff
@@ -1,8 +1,59 @@
...
+- Ubuntu 16.04
+- Ubuntu 18.04
+- Ubuntu 20.04
+- Ubuntu 22.04
+- MacOS 11
+- MacOS 12
+
+前往 [Releases](https://github.com/emqx/emqtt-bench/releases) 页面查看具体的`emqtt_bench`发布版本信息。
+
+例如，以下是如何在Ubuntu 20.04上安装 `emqtt_bench`:
```
Comment (LenaLenaPan) [inline zh_CN/performance/benchmark.md]:
```suggestion
例如，以下是如何在 Ubuntu 20.04 上安装 `emqtt_bench`:
```

File: zh_CN/performance/benchmark.md [hunk 16]
```diff
@@ -24,7 +75,7 @@ Usage: emqtt_bench pub | sub | conn [--help]
 
 输出以上内容，则证明 `emqtt_bench` 已正确安装到主机。
 
-## 使用
+## 用eMQTT-Bench进行性能测试
```
Comment (LenaLenaPan) [inline zh_CN/performance/benchmark.md]:
```suggestion
## 用 eMQTT-Bench 进行性能测试
```

**Review by LenaLenaPan — APPROVED** [review]

**Pull request merged**